crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -2349,7 +2342,7 @@ mod timed_tests {
         // Just checking the generic update is correct.
         assert_matches!(
             generic_stream.recv().await,
-            Ok(RoomEventCacheGenericUpdate::UpdateTimeline { room_id: expected_room_id }) => {
+            Ok(RoomEventCacheGenericUpdate{ room_id: expected_room_id }) => {
```

Comment:
```suggestion
            Ok(RoomEventCacheGenericUpdate { room_id: expected_room_id }) => {
```

Reply:
Ohnoes, rustfmt has given up on everything. thanks for catching those!